pymongo/pool.py

```diff
@@ -27,7 +27,7 @@
     HAS_SNI as _HAVE_SNI,
     IPADDR_SAFE as _IPADDR_SAFE)
 
-from bson import DEFAULT_CODEC_OPTIONS
+from bson import DEFAULT_CODEC_OPTIONS, codec_options
```

Comment:
Importing codec_options is not needed. Is your editor adding this automatically by accident?

Reply:
Yeah, I just disabled it